src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -807,5 +807,25 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 }
             }
         }
+
+#nullable enable
+        public sealed override MethodSymbol? PartialImplementationPart => _property is SourcePropertySymbol { IsPartialDefinition: true, OtherPartOfPartial: { } other }
+            ? (MethodKind == MethodKind.PropertyGet ? other.GetMethod : other.SetMethod)
+            : null;
+
+        public sealed override MethodSymbol? PartialDefinitionPart => _property is SourcePropertySymbol { IsPartialImplementation: true, OtherPartOfPartial: { } other }
+            ? (MethodKind == MethodKind.PropertyGet ? other.GetMethod : other.SetMethod)
+            : null;
+
+        internal bool IsPartialDefinition => _property is SourcePropertySymbol { IsPartialDefinition: true };
+        internal bool IsPartialImplementation => _property is SourcePropertySymbol { IsPartialImplementation: true };
+
+        public sealed override bool IsExtern
```

Comment:
This property is exposed publicly through `ISymbol.IsExtern`. Consider checking that public API from the `extern` tests you added